### ISSUE TITLE
Review contrast throughout site

### ISSUE THREAD

**Issue by greatislander** [fluid-project/infusion-docs#192]
## Describe the bug

As identified by @jobara (https://github.com/fluid-project/infusion-docs/issues/152#issuecomment-832181179), some content lacks an accessible contrast ratio. We should undertake a comprehensive review of contrast throughout the site, perhaps starting with an Axe audit.

**Comment by jobara** [comment]
It looks like some of the syntax highlighting may also not meet adequate contrast level standards. However, we will be replacing the existing system in #195 and could address any issues along with that work.
